synfig-studio/src/gui/mainwindow.cpp

```diff
@@ -194,6 +194,35 @@ MainWindow::init_menus()
 		sigc::ptr_fun(App::edit_custom_workspace_list)
 	);
 
+	//animation tabs
+	action_group->add( Gtk::Action::create("move-to-tab-1", Gtk::StockID("synfig-move-to-tab-1"), _("Move to Tab 1"), _("")),
```

Comment:
You can use `for` cycle here:
```
for (...) {
	action_group->add( Gtk::Action::create("move-to-tab-2", Gtk::StockID("synfig-move-to-tab-2"), _("Move to Tab 2"), _("")),
		sigc::bind<int>(sigc::mem_fun(*this, &MainWindow::move_tab), n)
	);
}
```

Also try to not use `Gtk::StockID` because it's already deprecated.

Reply:
yes and storing the names in a container sounds good will do.